tests/client_test.py

```diff
@@ -1169,6 +1169,68 @@ def test_get_price_history_every_minute_end_datetime(self):
             params=params)
 
 
+    @patch('tda.client.base.datetime.datetime', mockdatetime)
+    def test_get_price_history_every_minute_empty_extendedhours(self):
+        self.client.get_price_history_every_minute(
+            'AAPL', need_extended_hours_data=None)
+        params = {
+                'apikey': API_KEY,
+                'periodType': 'day',
+                # ONE_DAY
+                'period': 1,
+                'frequencyType': 'minute',
+                # EVERY_MINUTE
+                'frequency': 1,
+                'startDate': MIN_TIMESTAMP_MILLIS,
+                'endDate': NOW_DATETIME_PLUS_SEVEN_DAYS_TIMESTAMP_MILLIS,
+        }
+        self.mock_session.get.assert_called_once_with(
+            self.make_url('/v1/marketdata/AAPL/pricehistory'),
+            params=params)
+
+
+    @patch('tda.client.base.datetime.datetime', mockdatetime)
+    def test_get_price_history_every_minute_extendedhours(self):
+        self.client.get_price_history_every_minute(
+            'AAPL', need_extended_hours_data=True)
+        params = {
+                'apikey': API_KEY,
+                'periodType': 'day',
+                # ONE_DAY
+                'period': 1,
+                'frequencyType': 'minute',
+                # EVERY_MINUTE
+                'frequency': 1,
+                'startDate': MIN_TIMESTAMP_MILLIS,
+                'endDate': NOW_DATETIME_PLUS_SEVEN_DAYS_TIMESTAMP_MILLIS,
+                'needExtendedHoursData': True,
+        }
+        self.mock_session.get.assert_called_once_with(
+            self.make_url('/v1/marketdata/AAPL/pricehistory'),
+            params=params)
+
+
+    @patch('tda.client.base.datetime.datetime', mockdatetime)
+    def test_get_price_history_every_minute_no_extendedhours(self):
```

Comment:
Optional: I feel like you can remove these `no_extendedhours` tests, given that they really only test that the appropriate value is passed in to the implementation method and not that any real logic is run. Feel free to leave them in, though. 

I'll merge the PR once you resolve this comment.

Reply:
Yea, guess you're right since `extendedhours` already demonstrates successful functionality of the new feature and `empty_extendedhours` covers the other edge case. `no_extendedhours` appears redundant.